tests/bsim/bluetooth/mesh/src/test_beacon.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2021 Nordic Semiconductor
+ * Copyright (c) 2023 Nordic Semiconductor
```

Comment:
Please don't update copyright year. AFAIK it stays the same once the copyright is added